src/source_drivers/modbus/resources/point/point_singular.py

```diff
@@ -61,7 +67,63 @@ def delete(cls, uuid):
             point.delete_from_db()
         return '', 204
 
-# from flask_restful import Resource
+
+class ModbusPointPoll(ModbusPointBase):
+    @classmethod
+    @marshal_with(modbus_point_all_fields)
+    def get(cls, uuid: str):
+        point = ModbusPointModel.find_by_uuid(uuid)
+        if point is None:
+            abort(404, message=f'Modbus Point not found')
+        else:
+            event = EventCallableBlocking(ModbusPolling.poll_point, (point,))
+            EventDispatcher.dispatch_to_source_only(
+                event, MODBUS_SERVICE_NAME)
+            event.condition.wait()
+            if event.error:
+                abort(500, message=str(event.data))
+            else:
+                return event.data, 200
+
+
+class ModbusPointPollNonExisting(Resource):
+    parser = reqparse.RequestParser()
+    for attr in modbud_poll_non_existing_attributes:
+        parser.add_argument(attr,
+                            type=modbud_poll_non_existing_attributes[attr]['type'],
+                            required=modbud_poll_non_existing_attributes[attr].get('required', False),
+                            store_missing=False)
+
+    @classmethod
+    @marshal_with(point_store_fields)
+    def get(cls):
```

Comment:
Make it post instead of `GET` and can you share the JSON `body` example.

Reply:
Adding an example:

`POST`: `http://{{bac_rest_ip}}:{{bac_rest_port}}/api/modbus/poll/point`
```{
      "network_type": "RTU",
      "network_rtu_port": "/dev/ttyUSB0", 
      "device_address": 1,
      "point_register": 1,
      "point_register_length": 2,
      "point_function_code": "READ_INPUT_REGISTERS",
      "point_data_type": "FLOAT",
      "point_write_value": 0
  }```